files/python/install-pyenv.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eux -o pipefail
+
+# Install pyenv
+curl https://pyenv.run | bash
```

Comment:
Suggestion: For traceability, it'd be nicer to install from GitHub directly:

```suggestion
# Install pyenv
curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
```

Reply:
9f27446d064009d84451594018020a02c7b2c7d3